src/app/@ansyn/ol/plugins/annotations/annotations-context-menu/components/annotations-context-menu-buttons/annotations-context-menu-buttons.component.ts

```diff
@@ -64,6 +65,8 @@ export class AnnotationsContextMenuButtonsComponent implements OnInit, AfterView
 				}
 			})
 		);
+
+		this.isMetadataEnabled = this.olPluginsConfig.AnnotationsContextMenu.metadata.active;
```

Comment:
check for something like this this.olPluginsConfig?.AnnotationsContextMenu?.metadata?.active